__tests__/assertions.js

```diff
@@ -33,6 +33,16 @@ function expect(actual) {
         throw new Error(actual + " is not equal to " + expected);
       }
     },
+    toEqualArray: function(expected) {
```

Comment:
Is this function taken from jest source?
Would be good if we not introduce new functions. This would allow to use jest on the CI server.

Reply:
https://repl.it/repls/NormalEmptyClicks

It seems that we should correct our implementation of toEqual, as in jest it can easily compare arrays. Testing our implementation in InDesign doesn't pass.

Reply:
Totally. I just hacked that together. Really needs some love. :) Maybe I'll find some space at the weekend for it